package.json

```diff
@@ -53,6 +53,7 @@
     "font-awesome": "^4.7.0",
     "formik": "^0.10.5",
     "formsy-react": "^0.19.5",
+    "fsevents": "^1.1.3",
```

Comment:
I've been having trouble with my CPU getting pinned running deck (or running tests); this seems to fix it.